### DISCUSSION TITLE
"media is linked to an audiobookshelf server on a different address"

### DISCUSSION THREAD

**Discussion by The-Flix** [advplyr/audiobookshelf-app#1332] [Q&A] [answered]
Hi all, I am very happy with Adiobookshelf and the android app so far. And even happier since I changed my dynamic dns provider, since it is working a lot better now. But that made me some trouble and gives me messages like the above.

"media is linked to an audiobookshelf server on a different address (address). Progress will be synced when connected to this server address."

How may I change this? Any idea what and where I need to change, to have it all correctly running with the new dynamic dns provider? The abs server itself didn't change, only my nginx reverse proxy points a different url to the correct server now.

I hope anyone may have an idea.

Cheers
Felix

**Comment by nichwall** [comment, marked as answer]
You will need to delete the local copy and download it from the new URL for progress to by synced. The server is identified by the URL used to download the book. There are feature requests to improve this functionality.

**Reply by The-Flix** [reply]
Thanks a lot for the quick help! It indeed was the solution and helps me :-)
